src/index.tsx

```diff
@@ -46,29 +45,37 @@ const history = require('history').createBrowserHistory({
             callback(true);
         }
     },
-    basename: getContextPathBasename()
+    basename: getContextPathBasename(),
 });
 
 const logger = createLogger({
-    collapsed: true
+    collapsed: true,
 });
 
+if (erDev()) {
+    Sentry.init({
+        dsn: "https://f3482eab7c994893bf44bcb26a0c8e68@sentry.gc.nav.no/14",
```

Comment:
Siden logger sendes direkte til Sentry fra frontend må "nøkkelen" sjekkes inn i koden. Denne er ikke sensitiv, men det vil være mulig for alle som har nøkkelen å sende requester til prosjektet vårt.

Reply:
kunne nøkkelen vært plassert i vault? 💭 jeg vet ikke om vi bruker vault for frontend-appene

Reply:
Jeg vet ikke hvor mye det vil hjelpe, da nøkkelen uansett må hentes inn i frontend. Da blir den jo lesbar for bruker uansett.

Worst-case her er at noen spammer prosjektet med tull, så jeg tror det bør gå fint.